src/app/shared/components/footer/Footer-LinkList.component.tsx

```diff
@@ -89,6 +89,9 @@ export function FooterLinkListComponent(props: {
         );
     }
 
+    const apiDocumentationUrl =
+        "https://zoonotify-dev.bfr.berlin/cms/documentation/v1.0.0";
```

Comment:
Please do not hard-code the Server address.
All URLs are handled in the Routes file